src/DocumentFormat.OpenXml/System/Xml/Linq/XDocumentExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Xml.Linq
+{
+    internal static class XDocumentExtensions
+    {
+        public static void Save(this XDocument doc, Stream stream)
```

Comment:
Should we add the `SaveAsync` by XDocument.SaveAsync ?

Reply:
No, this is a polyfill for .NET 3.5. Nothing in the SDK is async. I've wondered if there are scenarios for that; it would be a pretty big undertaking. If you have scenarios where you'd see improvement with async, I'm happy to discuss it on a different issue.